src/analysis/convex/cone/proper.lean

```diff
@@ -113,7 +113,7 @@ instance : has_zero (proper_cone 𝕜 E) :=
 
 instance : inhabited (proper_cone 𝕜 E) := ⟨0⟩
 
-@[simp, norm_cast] lemma coe_zero : ↑(0 : proper_cone 𝕜 E) = (0 : convex_cone 𝕜 E) := rfl
```

Comment:
Why did you remove the `norm_cast`?

Reply:
Because [convex_cone.coe_zero](https://leanprover-community.github.io/mathlib_docs/analysis/convex/cone/basic.html#convex_cone.coe_zero) doesn't have one either.

Should I add it back? And add it to `convex_cone.coe_zero` as well?

Reply:
Yes. If in doubt, add `norm_cast`; it will give an error if the application makes no sense.